### ISSUE TITLE
UNMET PEER DEPENDENCY webpack@^2.2.1

### ISSUE THREAD

**Issue by yumetodo** [kagucho/entry-webpack-plugin#1]
```
> git clone https://github.com/kagucho/entry-webpack-plugin.git
Cloning into 'entry-webpack-plugin'...
remote: Counting objects: 17, done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 17 (delta 4), reused 16 (delta 3), pack-reused 0
Unpacking objects: 100% (17/17), done.

> cd entry-webpack-plugin

> npm install                                                                                            
entry-webpack-plugin@0.0.0 C:\Users\yumetodo\Documents\git\entry-webpack-plugin                          
+-- css-loader@0.26.1                                                                                    
| +-- babel-code-frame@6.22.0                                                                            
| | +-- chalk@1.1.3                                                                                      
| | | +-- ansi-styles@2.2.1                                                                              
| | | +-- escape-string-regexp@1.0.5                                                                     
| | | +-- has-ansi@2.0.0                                                                                 
| | | | `-- ansi-regex@2.1.1                                                                             
| | | +-- strip-ansi@3.0.1                                                                               
| | | `-- supports-color@2.0.0                                                                           
| | +-- esutils@2.0.2                                                                                    
| | `-- js-tokens@3.0.1                                                                                  
| +-- css-selector-tokenizer@0.7.0                                                                       
| | +-- cssesc@0.1.0                                                                                     
| | `-- regexpu-core@1.0.0                                                                               
| |   +-- regenerate@1.3.2                                                                               
| |   +-- regjsgen@0.2.0                                                                                 
| |   `-- regjsparser@0.1.5                                                                              
| |     `-- jsesc@0.5.0                                                                                  
| +-- cssnano@3.10.0                                                                                     
| | +-- autoprefixer@6.7.3                                                                               
| | | +-- browserslist@1.7.3                                                                             
| | | | `-- electron-to-chromium@1.2.2                                                                   
| | | +-- caniuse-db@1.0.30000624                                                                        
| | | +-- normalize-range@0.1.2                                                                          
| | | `-- num2fraction@1.2.2                                                                             
| | +-- decamelize@1.2.0                                                                                 
| | +-- defined@1.0.0                                                                                    
| | +-- has@1.0.1                                                                                        
| | | `-- function-bind@1.1.0                                                                            
| | +-- postcss-calc@5.3.1                                                                               
| | | +-- postcss-message-helpers@2.0.0                                                                  
| | | `-- reduce-css-calc@1.3.0                                                                          
| | |   +-- balanced-match@0.4.2                                                                         
| | |   +-- math-expression-evaluator@1.2.16                                                             
| | |   `-- reduce-function-call@1.0.2                                                                   
| | +-- postcss-colormin@2.2.2                                                                           
| | | `-- colormin@1.1.2                                                                                 
| | |   +-- color@0.11.4                                                                                 
| | |   | +-- clone@1.0.2                                                                                
| | |   | +-- color-convert@1.9.0                                                                        
| | |   | | `-- color-name@1.1.1                                                                         
| | |   | `-- color-string@0.3.0                                                                         
| | |   `-- css-color-names@0.0.4                                                                        
| | +-- postcss-convert-values@2.6.1                                                                     
| | +-- postcss-discard-comments@2.0.4                                                                   
| | +-- postcss-discard-duplicates@2.0.2                                                                 
| | +-- postcss-discard-empty@2.1.0                                                                      
| | +-- postcss-discard-overridden@0.1.1                                                                 
| | +-- postcss-discard-unused@2.2.3                                                                     
| | | `-- uniqs@2.0.0                                                                                    
| | +-- postcss-filter-plugins@2.0.2                                                                     
| | | `-- uniqid@4.1.1                                                                                   
| | |   `-- macaddress@0.2.8                                                                             
| | +-- postcss-merge-idents@2.1.7                                                                       
| | +-- postcss-merge-longhand@2.0.2                                                                     
| | +-- postcss-merge-rules@2.1.1                                                                        
| | | +-- caniuse-api@1.5.3                                                                              
| | | | +-- lodash.memoize@4.1.2                                                                         
| | | | `-- lodash.uniq@4.5.0                                                                            
| | | +-- postcss-selector-parser@2.2.2                                                                  
| | | | +-- flatten@1.0.2                                                                                
| | | | +-- indexes-of@1.0.1                                                                             
| | | | `-- uniq@1.0.1                                                                                   
| | | `-- vendors@1.0.1                                                                                  
| | +-- postcss-minify-font-values@1.0.5                                                                 
| | +-- postcss-minify-gradients@1.0.5                                                                   
| | +-- postcss-minify-params@1.2.2                                                                      
| | | `-- alphanum-sort@1.0.2                                                                            
| | +-- postcss-minify-selectors@2.1.1                                                                   
| | +-- postcss-normalize-charset@1.1.1                                                                  
| | +-- postcss-normalize-url@3.0.8                                                                      
| | | +-- is-absolute-url@2.1.0                                                                          
| | | `-- normalize-url@1.9.0                                                                            
| | |   +-- prepend-http@1.0.4                                                                           
| | |   +-- query-string@4.3.2                                                                           
| | |   | `-- strict-uri-encode@1.1.0                                                                    
| | |   `-- sort-keys@1.1.2                                                                              
| | |     `-- is-plain-obj@1.1.0                                                                         
| | +-- postcss-ordered-values@2.2.3                                                                     
| | +-- postcss-reduce-idents@2.4.0                                                                      
| | +-- postcss-reduce-initial@1.0.1                                                                     
| | +-- postcss-reduce-transforms@1.0.4                                                                  
| | +-- postcss-svgo@2.1.6                                                                               
| | | +-- is-svg@2.1.0                                                                                   
| | | | `-- html-comment-regex@1.1.1                                                                     
| | | `-- svgo@0.7.2                                                                                     
| | |   +-- coa@1.0.1                                                                                    
| | |   | `-- q@1.4.1                                                                                    
| | |   +-- colors@1.1.2                                                                                 
| | |   +-- csso@2.3.1                                                                                   
| | |   | `-- clap@1.1.2                                                                                 
| | |   +-- js-yaml@3.7.0                                                                                
| | |   | +-- argparse@1.0.9                                                                             
| | |   | | `-- sprintf-js@1.0.3                                                                         
| | |   | `-- esprima@2.7.3                                                                              
| | |   +-- mkdirp@0.5.1                                                                                 
| | |   | `-- minimist@0.0.8                                                                             
| | |   +-- sax@1.2.2                                                                                    
| | |   `-- whet.extend@0.9.9                                                                            
| | +-- postcss-unique-selectors@2.0.2                                                                   
| | +-- postcss-value-parser@3.3.0                                                                       
| | `-- postcss-zindex@2.2.0                                                                             
| +-- loader-utils@0.2.16                                                                                
| | +-- big.js@3.1.3                                                                                     
| | +-- emojis-list@2.1.0                                                                                
| | `-- json5@0.5.1                                                                                      
| +-- lodash.camelcase@4.3.0                                                                             
| +-- object-assign@4.1.1                                                                                
| +-- postcss@5.2.14                                                                                     
| | +-- js-base64@2.1.9                                                                                  
| | +-- source-map@0.5.6                                                                                 
| | `-- supports-color@3.2.3                                                                             
| |   `-- has-flag@1.0.0                                                                                 
| +-- postcss-modules-extract-imports@1.0.1                                                              
| +-- postcss-modules-local-by-default@1.1.1                                                             
| | `-- css-selector-tokenizer@0.6.0                                                                     
| +-- postcss-modules-scope@1.0.2                                                                        
| | `-- css-selector-tokenizer@0.6.0                                                                     
| +-- postcss-modules-values@1.2.2                                                                       
| | `-- icss-replace-symbols@1.0.2                                                                       
| `-- source-list-map@0.1.8                                                                              
+-- extract-text-webpack-plugin@2.0.0-rc.3                                                               
| +-- ajv@4.11.3                                                                                         
| | +-- co@4.6.0                                                                                         
| | `-- json-stable-stringify@1.0.1                                                                      
| |   `-- jsonify@0.0.0                                                                                  
| +-- async@2.1.4                                                                                        
| | `-- lodash@4.17.4                                                                                    
| `-- webpack-sources@0.1.4                                                                              
+-- html-loader@0.4.4                                                                                    
| +-- es6-templates@0.2.3                                                                                
| | +-- recast@0.11.22                                                                                   
| | | +-- ast-types@0.9.5                                                                                
| | | +-- esprima@3.1.3                                                                                  
| | | `-- private@0.1.7                                                                                  
| | `-- through@2.3.8                                                                                    
| +-- fastparse@1.1.1                                                                                    
| `-- html-minifier@3.3.1                                                                                
|   +-- camel-case@3.0.0                                                                                 
|   | +-- no-case@2.3.1                                                                                  
|   | | `-- lower-case@1.1.3                                                                             
|   | `-- upper-case@1.1.3                                                                               
|   +-- clean-css@4.0.7                                                                                  
|   +-- commander@2.9.0                                                                                  
|   | `-- graceful-readlink@1.0.1                                                                        
|   +-- he@1.1.1                                                                                         
|   +-- ncname@1.0.0                                                                                     
|   | `-- xml-char-classes@1.0.0                                                                         
|   +-- param-case@2.1.0                                                                                 
|   +-- relateurl@0.2.7                                                                                  
|   `-- uglify-js@2.7.5                                                                                  
|     +-- async@0.2.10                                                                                   
|     +-- uglify-to-browserify@1.0.2                                                                     
|     `-- yargs@3.10.0                                                                                   
|       +-- camelcase@1.2.1                                                                              
|       +-- cliui@2.1.0                                                                                  
|       | +-- center-align@0.1.3                                                                         
|       | | +-- align-text@0.1.4                                                                         
|       | | | +-- kind-of@3.1.0                                                                          
|       | | | | `-- is-buffer@1.1.4                                                                      
|       | | | +-- longest@1.0.1                                                                          
|       | | | `-- repeat-string@1.6.1                                                                    
|       | | `-- lazy-cache@1.0.4                                                                         
|       | +-- right-align@0.1.3                                                                          
|       | `-- wordwrap@0.0.2                                                                             
|       `-- window-size@0.1.0                                                                            
+-- style-loader@0.13.1                                                                                  
`-- UNMET PEER DEPENDENCY webpack@^2.2.1                                                                 
                                                                                                         
npm WARN entry-webpack-plugin@0.0.0 requires a peer of webpack@^2.2.1 but none was installed.            
npm WARN extract-text-webpack-plugin@2.0.0-rc.3 requires a peer of webpack@^2.2.0 but none was installed.

> npm run sample

> entry-webpack-plugin@0.0.0 sample C:\Users\yumetodo\Documents\git\entry-webpack-plugin
> cd sample; webpack

指定されたパスが見つかりません。

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files (x86)\\Nodist\\v-x64\\nodev7.4.0\\node.exe" "C:\\Program Files (x86)\\Nodist\\bin\\node_modules\\npm\\bin\\npm-cli.js" "run" "sample"
npm ERR! node v7.4.0
npm ERR! npm  v3.9.2
npm ERR! code ELIFECYCLE
npm ERR! entry-webpack-plugin@0.0.0 sample: `cd sample; webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the entry-webpack-plugin@0.0.0 sample script 'cd sample; webpack'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the entry-webpack-plugin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd sample; webpack
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs entry-webpack-plugin
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls entry-webpack-plugin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\yumetodo\Documents\git\entry-webpack-plugin\npm-debug.log
```

https://github.com/angular/angular-cli/issues/4539
に近い匂いを感じる

**Comment by akihikodaki** [comment]
> > npm run sample
> 
> > entry-webpack-plugin@0.0.0 sample C:\Users\yumetodo\Documents\git\entry-webpack-plugin
> > cd sample; webpack
> 
> 指定されたパスが見つかりません。
> 
> npm ERR! Windows_NT 10.0.14393
> npm ERR! argv "C:\\Program Files (x86)\\Nodist\\v-x64\\nodev7.4.0\\node.exe" "C:\\Program Files (x86)\\Nodist\\bin\\node_modules\\npm\\bin\\npm-cli.js" "run" "sample"
> npm ERR! node v7.4.0
> npm ERR! npm  v3.9.2
> npm ERR! code ELIFECYCLE
> npm ERR! entry-webpack-plugin@0.0.0 sample: `cd sample; webpack`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the entry-webpack-plugin@0.0.0 sample script 'cd sample; webpack'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the entry-webpack-plugin package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     cd sample; webpack
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs entry-webpack-plugin
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls entry-webpack-plugin
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     C:\Users\yumetodo\Documents\git\entry-webpack-plugin\npm-debug.log

That is a different problem. Can you open another issue for it?

**Comment by akihikodaki** [comment]
Please check if cd69a367e3f67183e532c144b2204abc41faa83c fixes the issue.

**Comment by akihikodaki** [comment]
The second problem was also fixed with commit f1611b4d5a1be08f248d0542be0d01ac3a9c80f3. Now I close this issue.
